cartridges/cloudinary_sfra_changes/cartridge/templates/default/product/components/variationAttribute.isml

```diff
@@ -66,12 +66,15 @@ Template Notes:
                 ${Resource.msg('label.select', 'common', null)} ${attr.displayName}
             </option>
             <isloop items="${attr.values}" var="attrValue">
-                <option value="${cldPageTypeQueryParam ? attrValue.url + cldPageTypeQueryParam : attrValue.url}" data-attr-value="${attrValue.value}"
-                    ${!attrValue.selectable ? 'disabled' : ''}
-                    ${attrValue.selected ? 'selected' : ''}
-                >
-                    ${attrValue.displayValue}
-                </option>
+                <isif condition="${cldPageTypeQueryParam}">
+                    <option value="${cldPageTypeQueryParam ? attrValue.url + cldPageTypeQueryParam : attrValue.url}" data-attr-value="${attrValue.value}" ${!attrValue.selectable ? 'disabled' : ''} ${attrValue.selected ? 'selected' : ''}>
```

Comment:
${cldPageTypeQueryParam ? attrValue.url + cldPageTypeQueryParam : attrValue.url} will always evaluate according to True, because of the condition in the line above